consortium/decisions/2024.06.05-Transition_to_Apero_Foundation as_Fiscal_Sponsor/answers.md

```diff
@@ -0,0 +1,141 @@
+
+## 6. What timeframe will the project/organization exist over, e.g., a set duration of time, periodically, open-ended, etc.?
+
+The Dat-Ecosystem operates on an open-ended basis, with projects evolving over time. Several member projects have existed for many years and are expected to continue contributing to the maintenance of the peer-to-peer open-source software stack. While some projects are short-lived and eventually wind down, such cases are the exception rather than the norm.
+
+## 7. Describe the resources and contractual obligations the applicant is brining into the sponsorship
+
+Presently, the Dat-Ecosystem funds amount to $3,807.60 USD and are held through [OpenCollective](https://opencollective.com/dat). The possibility exists for the transfer of the Dat brand and domains from Code for Science and Society. Additionally, an outstanding cryptocurrency donation is under the custody of our initial fiscal sponsor, Code for Science and Society, owing to the limitations of OpenCollective.com in managing crypto tokens.
+
+Furthermore, there are a few subscribed sponsors through Open Collective whose continued support we value. Therefore, it would be advantageous if the Apero Foundation could extend fiscal sponsorship through Open Collective. However, if such an arrangement is not feasible, we express our interest in fiscal sponsorship from the Apero Foundation irrespective of the platform.
+
+Currently, we have no other contractual obligations. Some minor expenses, such as a glitch.com hosting and the dat-ecosystem domain name are being held on behalf of dat-ecosystem by member projects. Periodically, funds received by the Dat-Ecosystem are allocated to reimburse these projects for the mentioned services.
+
+#### a. Describe the current project/organizational personnel and their status as employees, volunteers, or independent contractors.
+
+All personnell of Dat-Ecosystem are volunteers. Nobody is employed by Dat-Ecosystem. The personnel consists of consortium members only, each of which is also a (co-)founder of an independent Dat-Ecosystem member project. If the Dat-Ecosystem consortium decides to act in accordance with the manifesto, it usually assigns budgets to activities and selects people to carry out those activities and funds it via the budget it resides over.
+
+All personnel within the Dat-Ecosystem are volunteers; no individuals are employed by the Dat-Ecosystem. The personnel exclusively consist of consortium members, each of whom is also a (co-)founder of an independent Dat-Ecosystem member project. When the Dat-Ecosystem consortium decides to act in accordance with the manifesto, it typically allocates budgets to specific activities and appoints individuals to execute said activities, funding them through the respective budget under its oversight.
```

Comment:
Duplicated content

Reply:
Thanks for spotting this. Very confusing. No clue how that happened. Fixed it :-)